src/features/user-dashboard/components/UserDetailForm.tsx

```diff
@@ -112,13 +114,19 @@ export const UserDetailForm: FC<Props> = ({
           <Trans i18nKey='userDetail.accessHeading'>Access Information</Trans>
         </h5>
         <Form.Group className='mb-2' controlId='create-user-form-role'>
-          <Form.Label>{t('role', { ns: 'common' })}</Form.Label>
+          <Form.Label>
+            {t('role', { ns: 'common' })}
+            {isRoleSelectorDisabled && 
+                <p className='text-danger mb-0'>{t('userDetail.roleRestriction', { ns: 'translation' })}</p>
```

Comment:
Since `translation` is the first namespace registered on line 34, the namespace inclusion here is redundant and can be removed, which would keep it consistent with the rest of this file and other pages.

Reply:
Understood. I'll remove it. I needed to move that translation to the updateUser namespace anyways.

Reply:
Why does it need to be changed to updateUser? This is in the userDetailForm file so I think the current selection is correct.

Reply:
I see. I saw the "user profile" text and thought I made a mistake. I'll change it back. In the future, I think we may want to add "form" and "page" to the end of each namespace to make it more clear.

Reply:
@coreyshuman Okay. I reverted the change and removed the namespace argument. I'll merge it after I get an approval from @fluturecode.

Reply:
Agreed regarding your feedback on form/page. I took some notes on my confusing while going through the application on the inconsistent usage of view, form, page, and other terms. This would be a good area for refactor.